python/scripts/rfi_receiver.py

```diff
@@ -787,6 +789,73 @@ def http_server2():
     httpd.serve_forever()
 
 
+def rfi_zeroing():
+    # Wait until the correct UTC time (deals with daylight savings time)
+    t_now = datetime.utcnow()
+    t_transit = ephemeris.solar_transit(t_now)[0]
+    t_diff = datetime.utcfromtimestamp(t_transit) - t_now
+
+    # Wait until the correct UTC time (deals with daylight savings time)
+    print("Time until transit: {}".format(t_diff))
+    print("Sleeping for {} seconds".format(abs(t_diff.total_seconds())))
+
+    time.sleep(abs(t_diff.total_seconds()))
+
+    print("Waking up to zero RFI during solar transit")
+
+    downtime = 60  # minutes
+
+    # Endpoint parameters
+    url = "http://csBfs:54323/toggle-rfi-zeroing"
+    headers = {"content-type": "application/json", "Accept-Charset": "UTF-8"}
+
+    # Payload
+    payload = {"rfi_zeroing": False}
+    payload["rfi_zeroing"] = False
+
+    # Turn Off
+    on = True
+    try:
+        r = requests.post(url, data=json.dumps(payload), headers=headers)
+        if not r.ok:
+            print("RFI CRONJOB: Something went wrong in the request.")
+        else:
+            on = False
+            print("RFI CRONJOB: Request sent")
```

Comment:
Maybe `RFI CRONJOB` -> `RFI Solar Transit Toggle:`, or something like that.  This is no longer using `cron`

Reply:
I would recommend just using a Python logger object instead. Then you can give the logger a decent name, and it'll give that plus other context.